perllib/FixMyStreet/Cobrand/FixMyStreet.pm

```diff
@@ -39,6 +39,15 @@ sub restriction {
     return {};
 }
 
+sub munge_category_list {
+    my ($self, $options, $contacts, $extras) = @_;
+
+    # No TfL Traffic Lights category in Hounslow
+    my %bodies = map { $_->body->name => 1 } @$contacts;
+    return unless $bodies{'Hounslow Borough Council'};
+    @$options = grep { ($_->{category} || $_->category) ne 'Traffic Lights' } @$options;
```

Comment:
Looks like the category is "Traffic lights" on FixMyStreet so might be an idea to make this `!~ /traffic lights/i`